test/runtests.jl

```diff
@@ -1,11 +1,13 @@
 using Bijectors
 
 using ChainRulesTestUtils
+using ChangesOfVariables
 using Combinatorics
 using DistributionsAD
 using FiniteDifferences
 using ForwardDiff
 using Functors
+using InverseFunctions
```

Comment:
This shouldn't be needed, should it? Since `inverse` and `with_logabsdet_jacobian` are part of the API now I guess it would make sense to export them?

Reply:
>  I think we should re-export inverse and with_logabsdet_jacobian since they are part of the API now - or is there a specific reason for not exporting them?

I've been wondering about that myself - I have no strong opinion on it. In principle we don't have to re-export every API we implement methods for, of course - but then these function a central to Bijectors now.

Reply:
We're re-exporting the whole of Distributions, so ... :-)

Reply:
My main argument for exporting them is that they replace `forward` and `inv` which are exported and always available, respectively.

Reply:
Very true.

Ok, re-exports are added (last commit).